system_profile_archiver/delete.py

```diff
@@ -0,0 +1,21 @@
+import psycopg2
+
+from system_profile_archiver import config
+
+DELETE_SQL = "DELETE FROM historical_system_profiles WHERE inventory_id = %s"
+
+
+def delete_by_inventory_id(inventory_id):
```

Comment:
I think it would help debugging for this listener to spit out some logging messages that it got a request to delete host X and then that it succeeded or failed in deleting profiles for host X.

Reply:
that is a good idea, will add. thanks!